platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/fingerprint/ConfigurationCacheFingerprintChecker.kt

```diff
@@ -298,6 +301,25 @@ class ConfigurationCacheFingerprintChecker(private val host: Host) {
                         .text(" has been added")
                 }
             }
+
+            is ConfigurationCacheFingerprint.GradleProperty -> input.run {
+                // TODO:wip
```

Comment:
💭  I understand we intend to work on this futher immediately, but it still feels too hacky to merge a `TODO:wip` comment

Reply:
Tagged them properly